### ISSUE TITLE
Tomtom build seems broken

### ISSUE THREAD

**Issue by mvglasow** [navit-gps/navit#751]
Continuation of #737: An older Tomtom file I found had a config file named `tomtom480.xml` in `/navit/share`, whereas current CircleCI artifacts do not. If I SSH into the machine, I can find `./project/build/tomtom480.xml` but none of the XSLT transforms introduced in #737 seem to have been applied. In both cases, that goes for the (at the time) current build, the last build before I changed the Tomtom XSLT, and the latest build on master.

While I am not an expert on this platform, to me that looks like the Tomtom build is broken.

@lains had suggested this could have been introduced in #559, but tests indicate otherwise. A branch based on b077321 (the immediate parent of the changes in #559—just had to doctor the CircleCI config to force the Tomtom builds to run on CI) looks the same, no XML file in `/navit/share`.

I went back in CircleCI history as far as it would go (i.e. September 5, 2017) and already back then, the XML file was absent from the Tomtom builds. In other words, this file has been absent from the packages since we moved to CircleCI V2, that is, for almost a year and a half.

It seems most of the build is handled inside` build_tomtom_minimal.sh`, [there is a line there](https://github.com/navit-gps/navit/blob/7bd08d7ef699bcf51be0f11af99f62c6ce436bf4/scripts/build_tomtom_minimal.sh#L61) that copies the xml config, but strangely not in the $OUT_PATH directory as one would expect. It explains why the XML file does not show up in the output, but not why the XSLs of #737 are not getting applied.

Examining the commits for the build script did not reveal anything that looked suspicious to me, i.e. would have removed the XML config from the package.

@jkoan says in #737:

> But its true that all solutions for TomTom are really much work in progress. It needs to be fixed, i try to address this soon.

**Comment by lains** [comment]
Yes this is strange. If the build script has not been changed, where was the XML config file copied over to the output package?
On Android, the list of XSL files applied to the config are set using the cmake variable XSLTS (-DXSLTS=android is provided during build [here](https://github.com/navit-gps/navit/blob/bb34c13a582e1a6b13f3da7a342fc7eccb081b42/scripts/build_android.sh#L26))

**Comment by pgrandin** [comment]
I think that this was fixed in #875.
@jkoan i vote that we close?
